### PR TITLE
changed volunteer messaging to a bit less exclusive tone

### PR DESCRIPTION
**Pull request by yncyrydybyl**

### CODE REVIEW

File: templates/volunteer.html [hunk 1]
```diff
@@ -20,9 +20,9 @@ <h1>Volunteer for the Conference</h1>
         <div class="content-wrapper">
             <div class="content">
                 <div class="col">
-                    <p><span style="font-weight: 700;">2024-08-30 Update</span>: we cannot accept more volunteers for
+                    <p><span style="font-weight: 700;">2024-08-30 Update</span>: we cannot give out any more volunteers tickets for
                         this year! Thanks to everyone who signed
-                        up, and we're looking forward to expand the team next year.</p>
+                        up. If you already have a ticket and still want to help, reach out to us</p>
```
Comment (HarHarLinks) [inline templates/volunteer.html:25]: reach out where/how?

Comment (HarHarLinks) [inline templates/volunteer.html:25]:
```suggestion
                        up. If you already have a ticket and still want to help, reach out to us.</p>
```

**Review by HarHarLinks — CHANGES_REQUESTED** [review]

File: templates/volunteer.html (hunk 1 above)
Comment (yncyrydybyl) [reply templates/volunteer.html]: Via the methods mentioned in the footer. Currently mail, mastodon, linkedin.

**Review by thibaultamartin — APPROVED** [review]

File: templates/volunteer.html (hunk 1 above)
Comment (HarHarLinks) [reply templates/volunteer.html]: I think it would have been well to mention that explicitly... people are dumb and blind.

**Pull request merged**